requirements.txt

```diff
@@ -1,20 +1,15 @@
-# requirements.txt          for dependency installation via conda
-# docs/ports.txt            for dependency installation via mac-ports
-# setup.py                  for mintpy     installation via pip after the dependency installation above
 python>=3.6
```

Comment:
I think that this is the problem.
This is OK for `conda` but not for `pip`.
I understand that this file is also include via `pyptoject.toml` dynamic stuff but `pip` cannot resolve a package named `python`.

My recommendation is to remove `python` from the requirement file and maybe add directly to the CI configuration file creating the `conda` environment.

Reply:
That was the issue as @scottstanie has pointed out. I see that [circle CI has successfully installed python 3.10](https://app.circleci.com/pipelines/github/yunjunz/MintPy/2433/workflows/5726d71d-9396-45a7-a0a6-ce317e1e5dfa/jobs/2439?invite=true#step-103-15059_16) as well. So I guess we could remove this completely here.